pkg/reconciler/taskrun/resources/apply_test.go

```diff
@@ -860,6 +886,33 @@ func TestContext(t *testing.T) {
 				},
 			}},
 		},
+	}, {
+		description: "context taskRun namespace replacement with no defined namepsace in spec container",
```

Comment:
this description is the same as the one on line 868. Did you mean to make this one "with defined namespace" instead of "with no defined namespace"?